### ISSUE TITLE
Cannot read property ‘ownerDocument’ of null using 'cy.click'

### ISSUE THREAD

**Issue by wesmangum** [cypress-io/cypress#6324]
### Current behavior:

Calling `cy.click()` on a `div` element produces this error:

![image](https://user-images.githubusercontent.com/5713417/73786489-05068500-475f-11ea-9a0d-be3f639c3dd0.png)

Additional screenshots:
![Image 2020-02-04 at 3 09 20 PM](https://user-images.githubusercontent.com/5713417/73787246-80b50180-4760-11ea-86a8-db19ca030149.png)
![image](https://user-images.githubusercontent.com/5713417/73787230-772b9980-4760-11ea-9121-cdb427429431.png)

### Desired behavior:

`cy.click()` should be able to click on any div element. 

### Test code to reproduce

Test code:
```js
it('Sets alpha to 100% when choosing non-transparent values', () => {
  cy
      .get(`[data-automation-id="colorBox"]`)
      .should('be.visible')
      .click('center);   
});
```

HTML code: 
```html
<div data-automation-id="colorBox" mode="relative" value="[object Object]" threshold="0"
delay="0" style="position: relative; box-sizing: border-box; height: 150px; 
border: 1px solid rgb(33, 33, 33); 
border-radius: 2px; background: linear-gradient(rgba(0, 0, 0, 0), black), linear-gradient(to right, white, rgba(255, 255, 255, 0)) rgb(255, 0, 0); 
cursor: crosshair; width: 224px;" width="224">
  <div style="top: -6px; left: -6px; transform: translate3d(112px, 75px, 0px); 
  position: absolute; border-radius: 50%; box-sizing: border-box; border: 2px solid white; 
  width: 12px; height: 12px; 
  box-shadow: rgb(51, 51, 51) 0px 0px 0px 1px inset, rgb(51, 51, 51) 0px 0px 0px 1px;">
  </div>
</div>
```

### Versions

Cypress 3.8.3, macOS Catalina 10.15.3, Google Chrome Version 79.0.3945.130 (Official Build) (64-bit)

**Comment by jennifer-shehane** [comment]
I cannot recreate the issue in 3.8.3 in Chrome 80, Chromium 78, Electron 78 as described from the code provided. 

Please provide the other necessary code required to reproduce the issue. Make sure to only run the code you provide to us completely to ensure it recreates the issue.

The code I ran is below:

`index.html`

```html
<html>
<body>
  <div data-automation-id="colorBox" mode="relative" value="[object Object]" threshold="0" delay="0" style="position: relative; box-sizing: border-box; height: 150px; 
  border: 1px solid rgb(33, 33, 33); 
  border-radius: 2px; background: linear-gradient(rgba(0, 0, 0, 0), black), linear-gradient(to right, white, rgba(255, 255, 255, 0)) rgb(255, 0, 0); 
  cursor: crosshair; width: 224px;" width="224">
    <div style="top: -6px; left: -6px; transform: translate3d(112px, 75px, 0px); 
    position: absolute; border-radius: 50%; box-sizing: border-box; border: 2px solid white; 
    width: 12px; height: 12px; 
    box-shadow: rgb(51, 51, 51) 0px 0px 0px 1px inset, rgb(51, 51, 51) 0px 0px 0px 1px;">
    </div>
  </div>
</body>
</html>
```

`spec.js`

```js
it('Sets alpha to 100% when choosing non-transparent values', () => {
  cy.visit('index.html')
  cy.get(`[data-automation-id="colorBox"]`)
    .should('be.visible')
    .click('center')
})
```

![Screen Shot 2020-02-05 at 2 49 00 PM](https://user-images.githubusercontent.com/1271364/73823606-056a5300-4827-11ea-876b-e385cc295108.png)

**Comment by Bkucera** [comment]
closing since we cannot reproduce, leave a comment with a reproducible example and we can re-open this issue. Thanks!

**Comment by topaz-s** [comment]
Got this massage : "CypressError: Timed out retrying: Cannot read property 'ownerDocument' of null"
while calling cy.get on an element who suppose to change it color when it gets clicked.
the color have changed during the test but than the test stopped and sent the error massage.  

Test code to reproduce
Test code:
context('Interactions', () => {
    beforeEach(() => {
        cy.visit('https://demoqa.com/')
    })
    it('selectble', () => {
        cy.contains('Selectable').click();
        cy.get('#selectable').children().should('have.length', 7);
        cy.get('#selectable > :nth-child(5)').contains('5').click('center')
            .should('have.class', '.ui-selected')
    })
})

The website that I've been working on :
https://demoqa.com/selectable/

![Capture](https://user-images.githubusercontent.com/62712082/77669133-0c8f2100-6f8d-11ea-8f7e-1e69ac3c249b.PNG)

**Comment by jennifer-shehane** [comment]
Duplicate of #6923 which was released in [`4.4.1`](https://on.cypress.io/changelog#4-4-1).
